src/NServiceBus.Core/Recoverability/Recoverability.cs

```diff
@@ -25,10 +25,10 @@ public Recoverability()
                 "Message recoverability is only relevant for endpoints receiving messages.");
             Defaults(settings =>
             {
-                settings.SetDefault(SlrNumberOfRetries, DefaultSecondLevelRetryPolicy.DefaultNumberOfRetries);
-                settings.SetDefault(SlrTimeIncrease, DefaultSecondLevelRetryPolicy.DefaultTimeIncrease);
-
+                settings.SetDefault(SlrNumberOfRetries, DefaultNumberOfRetries);
+                settings.SetDefault(SlrTimeIncrease, DefaultTimeIncrease);
```

Comment:
nevermind, saw it. Can't we set a default here and override it in the `MoveFailedBlabla` (which we then can remove anyways?)